discovery/gossiper.go

```diff
@@ -2759,6 +2739,100 @@ func (d *AuthenticatedGossiper) handleChanAnnouncement(nMsg *networkMsg,
 		return nil, false
 	}
 
+	// TODO(roasbeef): this is a hack, needs to be removed after commitment
+	// fees are dynamic.
+	capacity.WhenSome(func(capacity btcutil.Amount) {
+		edge.Capacity = capacity
```

Comment:
As mentioned above, think we can just assign the values here without the `WhenSome`.